src/Common/PlatformLinux.inl

```diff
@@ -85,7 +85,7 @@ inline void _splitpath(const char* path, // Path Input
         )
 {
     if(!path)
-        return EINVAL;
+        return;
```

Comment:
This case is defined in MSDN in the following way:

> If path is NULL, the invalid parameter handler is invoked, as described in Parameter Validation. If execution is allowed to continue, errno is set to EINVAL and the function returns EINVAL.

We're trying to reproduce that second case. Notice that it doesn't make sense, because _splitpath is supposed to return `void`. Maybe it should insert `errno = EINVAL`.

Reply:
Yes, reporting an error is an alternative solution. If this function tries to mimic the Windows implementation - I agree that it's the preferrable way.